examples/v1alpha1/pipelineruns/pipelinerun.yaml

```diff
@@ -92,7 +92,7 @@ spec:
       type: image
   steps:
   - name: build-and-push
-    image: gcr.io/kaniko-project/executor:v0.17.1
+    image: gcr.io/kaniko-project/executor:latest
```

Comment:
Can we keep this at a specific tag, just pick a more recent one?

Reply:
done